fleet_adapter_mir/MiRCommandHandle.py

```diff
@@ -660,123 +662,42 @@ def load_mir_positions(self):
     ##########################################################################
     # MISSION METHODS
     ##########################################################################
-    def queue_move_coordinate_mission(self, mir_location):
-        """Add a move mission to the mission queue, creating when needed."""
-        mission_name = ('move_coordinate_to'
-                        f'_{mir_location.x:.3f}'
-                        f'_{mir_location.y:.3f}'
-                        f'_{mir_location.yaw:.3f}')
-
-        # Get mission GUID. If missing, create one and save it.
-        mission_id = self.mir_missions.get(
-            mission_name, self.create_move_coordinate_mission(mir_location)
-        )
-
-        # Queue mission
+    def queue_variable_move_coordinate_mission(self, mir_location):
+        variable_move_mission_guid = \
+            self.mir_missions[self.mir_variable_move_mission]['guid']
+        mission = {
+            'mission_id': variable_move_mission_guid,
+            'parameters': [
```

Comment:
We could consider adding all of the path waypoints to this mission, and then we don't need to have the `_path_following_thread` anymore. The only catch is that we'd need to be able to track MiR's progress towards each waypoint, and I don't know if MiR's API provides that level of detail.

We can open an issue about this to work on it in the future.

Reply:
I might be mistaken but I don't believe I saw any support for move actions composed of variable number of coordinates. I can't find any definitive documentation on the set of actions available, but this screenshot from this [document](https://cdn.kyklo.co/assets/W1siZiIsIjIwMTkvMTAvMTYvMTgvNTgvNTYvMGNkOTJiZGQtZGYzMS00OTVmLThlMGEtMWUwODhiNzQ1ODgwL01pUiUyMC0lMjBNaVIlMjByb2JvdCUyMHJlZmVyZW5jZSUyMGd1aWRlLnBkZiJdXQ?sha=9b51843b3f3b3c7f), seems to show that such a feature is missing.
![image](https://user-images.githubusercontent.com/5383623/188857771-23c66876-5d44-484d-94a0-4e0bb4fc98b9.png)

We will probably need to continue sending out move commands waypoint by waypoint